Sources/Common/Util/CleanupRepository.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public protocol CleanupRepository: AutoMockable {
```

Comment:
had to be shifted to `common` so we can hold a strong reference in CustomerIO class which is also is `common`. Implementation lives in tracking.